### PR TITLE
Lint tasks.py

### PR DESCRIPTION
**Pull request by qinqon**
**Is this a BUG FIX or a FEATURE ?**:
/kind cleanup

**What this PR does / why we need it**:
The tasks.py do not adhere to any python lint standard, this change adhere with the commonly used "black" linter.

```release-note
Not user facing change: lint tasks.py
```

### CODE REVIEW

**Comment by fedepaol** [comment]
Also, changed the release note description as not user facing.

File: website/content/community/_index.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ To develop MetalLB, you'll need a couple of pieces of software:
 - [kind](https://github.com/kubernetes-sigs/kind), a lightweight Kubernetes cluster running in Docker
 - [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the Kubernetes commandline interface
 - [Invoke](https://www.pyinvoke.org) to drive the build system
+- [Black](https://github.com/psf/black) to lint python code
```
[the remark this answers is not in the document]
Comment (qinqon) [reply website/content/community/_index.md]: Ahh wait, there is already a requirements.txt `https://github.com/metallb/metallb/blob/main/dev-env/requirements.txt` we should just add black there.

**Comment by qinqon** [comment]
Reformated latest task.py, with latest from main.

File: tasks.py [hunk 2]
```diff
@@ -792,6 +792,12 @@ def test(ctx):
     run("KUBEBUILDER_ASSETS={} go test -short -race ./...".format(kubebuilder_assets), echo=True)
 
 
+@task
+def linttasks(ctx):
+    """Check if 'tasks.py' has being correctly formatted with 'black' linter."""
```
Comment (fedepaol) [inline tasks.py]: Can we have this command perform the actual lint instead of just checking? It's going to be easier to know what to fix.

Comment (qinqon) [reply tasks.py]: since the fix is just about calling "black", maybe what we need is CI failing on "missing linting task.py with `black tasks.py`"

Comment (fedepaol) [reply tasks.py]: right, which is what the `inv checkchanges` execution is supposed to do:

Just add 
`inv checkchanges --action="run inv linttasks"`
after the execution.

If checkchanges is not doing what is supposed to do, we'll file an issue to take care of it, but I'd keep the same format for ci: apply the changes, check that nothing changed.

Comment (qinqon) [reply tasks.py]: Ahh cool, then I will do that thanks!

File: tasks.py [hunk 3]
```diff
@@ -792,6 +792,12 @@ def test(ctx):
...
+    run("black --check tasks.py")
```
Comment (fedepaol) [inline tasks.py]: now this must lint, not only check, right?

Comment (fedepaol) [reply tasks.py]: nevermind, I saw you changed it later. Let me squash on merge

**Pull request merged**